packages/aws.rb

```diff
@@ -3,57 +3,31 @@
 class Aws < Package
   description 'The AWS CLI is an open source tool built on top of the AWS SDK for Python (Boto) that provides commands for interacting with AWS services.'
   homepage 'https://aws.amazon.com/documentation/cli/'
-  version '1.17.13'
+  version '0.2.7'
```

Comment:
Shouldn't the version be `2.0.2.7`?

Reply:
maybe the package name should be aws2 and the version 0.2.7. That's what it looks like to me

Reply:
Technically the package name should be AWS2, version 0.2.7:
https://pypi.org/project/aws2/

Reply:
Actually, there are two separate packages:
https://pypi.org/project/aws2/#history
https://pypi.org/project/aws/#history

Which one is this supposed to be?

Reply:
Looks like we want aws2...

Reply:
(aws is the older, abandoned project)